<launch>
  
  <!-- ARGS -->
  <arg name="degrees" default="false" />
  <arg name="topic_str" default="/mavros_ned/estimate" />

  <!-- Euler Printer -->
  <node name="euler_angle_printer" pkg="ibvs_sim" type="print_euler_angles.py" output="screen">

    <param name="is_in_degrees" value="$(arg degrees)" />
    <param name="topic_string" value="$(arg topic_str)" />

  </node>

</launch>
